docs/user-guide/home/global-modules/README.md

```diff
@@ -0,0 +1,134 @@
+
+![openproject_user_guide_global_work_packages_single_view](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_work_packages_full_view.png)
+
+## Calendars
+
+**Calendars** global module displays all calendars that you have rights to see and all public calendars. 
+
+![openproject_user_guide_global_calendars](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_calendars.png)
+
+You can also create a new calendar directly from the global modules menu by clicking the green **+Calendar** button. 
+
+![openproject_user_guide](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_add_calendar.png)
+
+Here you can name the calendar, select a project to which a calendar should be assigned to and set to be public or favored. Find out more about editing calendars [here](../../calendar). 
+
+## Team planners
+
+**Team planners** global module will display all team planners from the projects you are a member of, have administrative privileges to see and the public ones. 
+
+![openproject_user_guide_global_team_planners](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_team_planner.png)
+
+You can also create a new team planner directly from the global modules menu by clicking the green **+Team planner** button. 
+
+![openproject_user_guide_add_new_team_planner](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_add_team_planner.png)
+
+Here you can name the team planner, select a project to which a team planner should be assigned to and set to be public or favored. Find out more about editing team planners [here](../../team-planner). 
+
+## Boards
+
+**Boards** global module will list all boards that have been created in the projects you are a member of, have administrative privileges to see or public projects. 
+
+![openproject_user_guide_global_boards](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_boards.png)
+
+You can also create a board directly from the global modules menu by clicking the green **+Board** button. 
+
+![openproject_user_guide_add_new_board](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_add_board.png)
+
+Here you can name the board, select a project to which a board should be assigned to and choose the board type. Find out more about editing boards [here](../../agile-boards). 
+
+## News
+
+**News** global module will display all news that have been published in projects you are a member of, have administrative privileges to see and public projects. 
+
+TIMEFRAME - ?
```

Comment:
As far as I know there is no timeframe for news, it is a paginated view until the very first new created.